btree.go

```diff
@@ -308,6 +308,10 @@ func (b btreeIndex) SizeBytes() (sz int) {
 // 3. Binary search the bucket (in MEM)
 // 4. Return the simple section pointing to the posting list (in MEM)
 func (b btreeIndex) Get(ng ngram) (ss simpleSection) {
+	if b.bt == nil {
+		return simpleSection{}
+	}
```

Comment:
is this fixing a bug in main?

Edit: I think it is from ngramFrequency were for ngram we handle it being nil but in the new code you don't do it there?

Reply:
This is just in anticipation of us using the concrete type IE btreeIndex in indexdata, in which case a well behaved 0-value is a nice-to-have for tests. In production we always contruct the btreeIndex with newBtreeIndex which sets the btree correctly.